### DISCUSSION TITLE
Issues with pushing a first branch to github

### DISCUSSION THREAD

**Discussion by ghost** [community/community#110638] [New to GitHub]
IM attempting to push a first repo to github, and cannot for some reason. Adding the repo in github desktop puts it under the name of an old project (which would have had the same local path but is a different project), and then attempting to push the repo gives the following generic error: _The repository does not seem to exist anymore. You may not have access, or it may have been deleted or renamed._

I did delete the old project out of my web github before attempting to reupload, but it is still visible in deleted projects.

**Comment by AidenJamesYt** [comment]
github actions, pushing, and issues are all currently experiencing degraded performance, meaning it is down for some people. To stay updated on the status, [visit the GitHub Status Site](https://www.githubstatus.com/) for live status updates.

**Comment by ghost** [comment]
Adding that changing the name and path of the local repo ALSO pulls the old repo name on github

**Comment by ghost** [comment]
Unfortunately it seems that status returning to normal has not fixed this issue. This new project is still directly associating with the old and making it impossible to push

**Comment by ghost** [comment]
Attempting to add the repo to the origin via command line, the error is "origin already exists" which it doesn't, and then it asks for password authentication which was deprecated in 2021.

**Comment by ghost** [comment]
Even better, github desktop is so screwed that signing out and signing in with a fully different account associates any repo added with the OLD DELETED REPO from this account. What a fabulous ecosystem!
